modules/ROOT/pages/install-odatav4-tooling.adoc

```diff
@@ -0,0 +1,24 @@
+= Installing OData V4 Tooling
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+To install APIkit for OData v4 plugin in Anypoint Studio:
+
+. Select *Help* > *Install New Software* > *Add*.
++
+image::add-update-site.png["Window for installing new software with the *Add* button highlighted"]
+. In *Name*, type `APIkit for ODATA v4 Update Site`.
+. In *Location*, type the URL of the site: `+https://studio.mulesoft.org/s4/apikit-for-odata4/+`.
+. Select *Add*.
```

Comment:
Use "click" for buttons.
https://confluence.internal.salesforce.com/display/MTDT/MuleSoft+CX+Writing+Style+Reference#MuleSoftCXWritingStyleReference-%22click%22

Reply:
OK! Thanks!